Src/Support/Google.Apis.Auth/OAuth2/GoogleAuthConsts.cs

```diff
@@ -75,6 +75,21 @@ public static class GoogleAuthConsts
         /// <summary>Installed application localhost redirect URI.</summary>
         public const string LocalhostRedirectUri = "http://localhost";
 
+        /// <summary>IAM access token endpoint for service account.</summary>
```

Comment:
These don't look like they should be public - at least not the ones with the `{0}` in.

I'd prefer to keep them internal for now, and only make anything public if we really need to. Where we've got format strings, if we need to make those public I'd probably do so via a method.

Reply:
done